File: src/Shuffle The Array/problem.cpp
class Solution {
public:
    vector<int> shuffle(vector<int>& nums, int n) {
        vector<int> shuffledVector(2*n, 0);
        int i = 0;
        int j = n;
        int count = 0;
        for(i = 0; i < n; i++, j++)
        {
            shuffledVector[count] = nums[i];
            shuffledVector[count + 1] = nums[j];
            count += 2;
        }
        return shuffledVector;
    }
};

File: src/Count Items Matching A Rule/problem.py
class Solution:
    def countMatches(self, items: List[List[str]], ruleKey: str, ruleValue: str) -> int:
        count = 0
        if ruleKey == "type":
            for i in range(0, len(items)):
                if ruleValue == items[i][0]:
                    count += 1
        elif ruleKey == "color":
            for i in range(0, len(items)):
                if ruleValue == items[i][1]:
                    count += 1
        else:
            for i in range(0, len(items)):
                if ruleValue == items[i][2]:
                    count += 1
        return count
